data/flows/patterns.yml

```diff
@@ -51,4 +51,31 @@ flows:
     name: pattern completed
     steps:
       - action: utter_can_do_something_else
-      - action: action_reset_routing
\ No newline at end of file
+      - action: action_reset_routing
+
+  pattern_clarification:
+    description: Conversation repair flow for handling ambiguous requests that could match multiple flows
+    name: pattern clarification
+    steps:
+      - action: action_clarify_flows
+        next:
+          - if: context.names contains "add a contact"
+            then:
+              - link: add_contact
+          - else: clarify_options
+      - id: clarify_options
+        action: utter_clarification_options_rasa
+
+  pattern_clarification_with_human_handoff:
```

Comment:
Is this a new pattern added to the default file in rasa-private? Or should this be overriding pattern handoff?

Reply:
iiuc this is a new pattern added to `rasa-private` recently in #966

Reply:
This file on `main` doesn't contain such pattern, so Rasa will not classify it as a pattern, but as a user defined flow: https://github.com/RasaHQ/rasa-private/blob/main/rasa/dialogue_understanding/patterns/default_flows_for_patterns.yml

Reply:
I looked at the PR and the changelog mentions that `pattern_human_handoff` can now be linked in other flows, therefore this flow should be overriding that pattern name.